camel-idea-plugin/src/main/resources/META-INF/plugin.xml

```diff
@@ -47,6 +47,7 @@
 
   <extensions defaultExtensionNs="com.intellij">
     <notificationGroup id="Apache Camel" displayType="BALLOON"/>
+    <postStartupActivity implementation="com.github.cameltooling.idea.CamelPluginStartup"/>
```

Comment:
What is the benefit of it versus what we currently do?

Reply:
`CamelPluginStartup` was using a deprecated method (`projectOpened`).